app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,50 @@
+$( "a.upvote i" ).click(function( event ) {
+  event.preventDefault();
+  event.stopPropagation();
+  $(this).toggleClass("gold");
+  var reviewId = $(this).attr("id").match(/review-([0-9]+)/)[1];
```

Comment:
The general idea makes sense, but I'm confused about how it applies in this situation. Here I'm just trying to grab the id number on the tail end of this attribute's id so I know which upvote or downvote number to change the color of. I'm not sure how I could accomplish this with getElementById since (id name) would be unknown without this line?